packages/uniforms/__tests__/GraphQLBridge.js

```diff
@@ -239,6 +244,34 @@ describe('GraphQLBridge', () => {
         it('works with label (falsy)', () => {
             expect(bridgeI.getProps('id', {label: null})).toEqual({
                 label: '',
+                placeholder: 'Post ID',
+                required: true,
+                allowedValues: [1, 2, 3]
+            });
+        });
+
+        it('works with placeholder (custom)', () => {
+            expect(bridgeI.getProps('id', {placeholder: 'Post ID'})).toEqual({
+                label: 'Post ID',
+                placeholder: 'Post ID',
+                required: true,
+                allowedValues: [1, 2, 3]
+            });
+        });
+
+        it('works with placeholder (true)', () => {
+            expect(bridgeI.getProps('id', {placeholder: true})).toEqual({
+                label: 'Post ID',
+                placeholder: 'Post ID',
+                required: true,
+                allowedValues: [1, 2, 3]
+            });
+        });
+
+        it('works with placeholder (falsy)', () => {
+            expect(bridgeI.getProps('id', {placeholder: null})).toEqual({
+                label: 'Post ID',
+                placeholder: '',
```

Comment:
Test for `placeholder === true` and `extra.placeholder === undefined` is missing.